ksqldb-parser/src/test/java/io/confluent/ksql/util/QueryMetaDataCollectorTest.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class QueryMetaDataCollectorTest {
+  private KsqlConfig ksqlConfig;
+
+  @Before
+  public void setUp() {
+    // initialize ksql config
+    Map<String, String> ksqlConfigMap = new HashMap<>();
+    ksqlConfigMap.put(KsqlConfig.KSQL_CCLOUD_QUERYANONYMIZER_CLUSTER_NAMESPACE,
```

Comment:
see above re first arg: then you don't have to do this complicated mocking, just pass the literal around and your testing becomes easier